src/components/modals/location/Delete.vue

```diff
@@ -27,7 +54,8 @@ export default {
   name: "DeleteLocationModal",
   props: {
     name: {
-      type: String
+      type: String,
+      default: '',
```

Comment:
Please check this line, the linter required a default value here, but I wasn't sure what to set (especially in german :smile: )

Reply:
I believe this can be solved by removing the `default` and replacing it with `required: true`